proto/blueprint.proto

```diff
@@ -369,10 +372,33 @@ message Topic {
   string name = 2;
 }
 
+message MyGuardianFollow {
+  enum FollowType {
+    FOLLOW_TYPE_UNSPECIFIED = 0;
+    FOLLOW_TYPE_CONTRIBUTOR = 1;
+    FOLLOW_TYPE_KEYWORD = 2;
+    FOLLOW_TYPE_SERIES = 3;
+    FOLLOW_TYPE_SECTION = 4;
+  }
+  string id = 1;
+  string web_title = 2;
+  FollowType type = 3;
+}
+
```

Comment:
I just moved this up the file a bit to better represent that this is not part of tracking.